src/Repository/CitizenProjectCommentRepository.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+namespace AppBundle\Repository;
+
+use AppBundle\Entity\Adherent;
+use AppBundle\Entity\CitizenProject;
+use Doctrine\ORM\EntityRepository;
+
+class CitizenProjectCommentRepository extends EntityRepository
+{
+    public function findCitizenProjectComments(CitizenProject $citizenProject): array
```

Comment:
I would call it `findForProject`, we already know we will find ...comments thanks to the ...CommentRepository.

Here you can add useful phpdoc like `@return ...Comment[]` to avoid duck type hinting in the code.

Reply:
Agree